src/bin/console/pagination.cpp

```diff
@@ -105,6 +106,37 @@ void Pagination::display() {
         break;
       }
 
+      case KEY_RIGHT: {
+        ++current_column;
+        reprint = true;
+        break;
+      }
+
+      case KEY_LEFT: {
+        if (current_column > 0) {
+          --current_column;
```

Comment:
Having had the chance to test this, I'd rather make this 5 or so. What do you think?